pkg/catalog/catalog.go

```diff
@@ -2020,6 +2005,14 @@ func (c *Catalog) GetAddressToken(ctx context.Context, repository, token string)
 	return c.Store.GetAddressToken(ctx, repo, token)
 }
 
+func (c *Catalog) DeleteExpiredAddressTokens(ctx context.Context, repository string) error {
```

Comment:
As far as I understand this is not used in the scope of this PR and should move to:
https://github.com/treeverse/lakeFS/pull/4936

Reply:
#4936 is for the delete job. To test it I need to use Set and Get that's why I need it in this PR

Reply:
I think there's a misunderstanding all the DeleteExpiredAddressTokens functions (catalog and graveler) are not really used in this PR and should move to the #4936 as this is the right context for them (correct me if I'm wrong)